mindsdb/api/mysql/mysql_proxy/datahub/datanodes/mindsdb_datanode.py

```diff
@@ -1,17 +1,12 @@
 import json
-import copy
-from datetime import datetime
 
-from lightwood.api import dtype
 import pandas as pd
 import numpy as np
 
 from mindsdb.api.mysql.mysql_proxy.datahub.datanodes.datanode import DataNode
 from mindsdb.api.mysql.mysql_proxy.utilities.sql import query_df
-from mindsdb.api.mysql.mysql_proxy.utilities.functions import get_column_in_case
-from mindsdb.utilities.functions import cast_row_types
 from mindsdb.utilities.config import Config
-from mindsdb.api.mysql.mysql_proxy.utilities import SqlApiException
+from mindsdb.api.mysql.mysql_proxy.utilities.lightwood_dtype import dtype
```

Comment:
This is still being used in `_get_model_columns`, but we should make it so that it calls the respective handler method instead, because having Lightwood `dtypes` here goes against mutual independence.